woocommerce/woocommerce-ios
# [Mobile Payments] Enable Tap to Pay on iPhone Milestone 2 feature flag

Part of: #9370 
Merge after #9516 


## Description


This PR enables the feature flag for Milestone 2 of sTAP Away, the Tap to Pay on iPhone project, which adds tools for promoting the set up of TTPoI.

With this flag on:
- Universal links to `https://woocommerce.com/mobile/payments/tap-to-pay` open the set up flow.
- JITMs with CTAs to that link, open the set up flow.
- In Person Payments onboarding is shown before the Set up Tap to Pay on iPhone flow, when required.

## Testing instructions


Make a release build of the app to run on a simulator
Verify that when you open a deep link to `https://woocommerce.com/mobile/payments/tap-to-pay` it shows the Tap to Pay on iPhone set up flow (Reminders is a good app to paste the link in to)

---
